charts/ist-loader/values.yaml

```diff
@@ -0,0 +1,8 @@
+istLoader:
+  image:
+    # istLoaderLoader.image.repository -- Docker image
```

Comment:
The description is not showing up in the generated docs because it says `istLoaderLoader` instead of `istLoader`. probably a typo.